dockertest/config.py

```diff
@@ -203,6 +203,7 @@ def merge_write(self, fileobject):
         scp = SafeConfigParser()
         # Safe if file doesn't exist
         scp.read(fileobject.name)
+        # N/B: This won't work with DEFAULTS
```

Comment:
Do you know why? If so can you explain it in the comment? I really don't understand this. `ConfigParser` special-cases DEFAULT (singular), but I don't see why the plural should also be affected.